code/game/objects/items/storage/boxes.dm

```diff
@@ -34,6 +34,23 @@
 	w_class = SIZE_LARGE //Changed becuase of in-game abuse
 	storage_flags = STORAGE_FLAGS_BOX
 
+/obj/item/storage/box/pride
```

Comment:
This could inherit `/obj/item/storage/fancy/crayons` instead if you implement the functionality where the crayons appear in the overlay (in otherwords each crayon by name needs to have an icon_state in crayons.dmi like how red, orange, etc are to appear in the box).

Reply:
I would've had to add some more sprites to add the functionality of the crayons appearing inside of the box sprite, but I was exhausted when making this PR and I wanted to get it out ASAP for pride! I could go back and touch it up buuuut I'm busy right now. If I have a moment in a bit I could :p